src/azure-cli/azure/cli/command_modules/vm/_help.py

```diff
@@ -3384,3 +3384,127 @@
+    examples:
+      - name: Pause executing next line of CLI script until the restore-point is successfully created.
+        text: |-
+               az restore-point wait --resource-group "myResourceGroup" --collection-name "rpcName" \
+--name "rpName" --created
+      - name: Pause executing next line of CLI script until the restore-point is successfully deleted.
+        text: |-
+               az restore-point wait --resource-group "myResourceGroup" --collection-name "rpcName" \
+--name "rpName" --deleted
+"""
+
+helps['restore-point collection'] = """
+    type: group
+    short-summary: Manage restore point collection
+"""
+
+helps['restore-point collection list'] = """
+    type: command
+    short-summary: "Gets the list of restore point collections in a resource group."
+    examples:
+      - name: Gets the list of restore point collections in a resource group.
+        text: |-
+               az restore-point collection list --resource-group "myResourceGroup"
+"""
+
+helps['restore-point collection show'] = """
+    type: command
+    short-summary: "Get the restore point collection."
+    examples:
+      - name: Get a restore point collection (but not the restore points contained in the restore point collection)
+        text: |-
+               az restore-point collection show --resource-group "myResourceGroup" --collection-name "myRpc"
+      - name: Get a restore point collection, including the restore points contained in the restore point collection
+        text: |-
+               az restore-point collection show --resource-group "myResourceGroup" --collection-name "rpcName"
+"""
+
+helps['restore-point collection create'] = """
+    type: command
+    short-summary: "Create the restore point collection. Please refer to https://aka.ms/RestorePoints \
+for more details. When updating a restore point collection, only tags may be modified."
+    examples:
+      - name: Create or update a restore point collection.
+        text: |-
+               az restore-point collection create --location "norwayeast" --source-id "/subscriptions/{subscription-id}\
+/resourceGroups/myResourceGroup/providers/Microsoft.Compute/virtualMachines/myVM" --tags myTag1="tagValue1" \
+--resource-group "myResourceGroup" --collection-name "myRpc"
+"""
+
+helps['restore-point collection update'] = """
+    type: command
+    short-summary: "Update the restore point collection."
+"""
+
+helps['restore-point collection delete'] = """
+    type: command
+    short-summary: "Delete the restore point collection. This operation will also delete all the \
+contained restore points."
+"""
+
+helps['restore-point collection list-all'] = """
+    type: command
+    short-summary: "Gets the list of restore point collections in the subscription. Use nextLink property in the \
+response to get the next page of restore point collections. Do this till nextLink is not null to fetch all the restore \
+point collections."
```

Comment:
Please use the active voice in the help description
```suggestion
    short-summary: "Get the list of restore point collections in the subscription. Use nextLink property in the \
response to get the next page of restore point collections. Do this till nextLink is not null to fetch all the restore \
point collections."
```